docs/content/manual/pre-release/upgrade/auto-upgrade-engine.md

```diff
@@ -13,9 +13,17 @@ title: Automatically Upgrading Longhorn Engine Test
 1. Create a StatefulSet of scale 10 using 10 Longhorn volume. Set node selector so that all pods land on the same node.
 2. Upgrade Longhorn to use a newer default engine image
 3. In Longhorn UI and Longhorn manager logs, Verify that Longhorn doesn't upgrade all volumes at the same time. Only 3 at a time.
-
+4. In longhorn UI, while the concurrent value is set to limit 1, change the value to higher number and verify if the upgrade is happening as per the new value.
+5. Create stateful set pods with volume claim template, set the scale to 10 and in the configure options, select RWO and RWX. Verify the volumes created have RWO set
+6. While the auto upgrade is set, try to manually upgarde the image engine to the latest version.
+7. Take backup for a volume when upgarde hasn't started
+8. Create 10 volmes and set concurrent upgrade limit to 2. When the upgrade starts, delete all the volumes.
```

Comment:
What is the expectation?
Is it correct that volumes are deleted and Longhorn system is not corrupted?

Reply:
Yes, the volumes are deleted and the longhorn system is not corrupted when I tried.

Reply:
`Volumes` typo